test/JDBC/expected/sys-all_sql_modules-vu-prepare.out

```diff
@@ -38,6 +38,13 @@ CREATE PROC sys_all_sql_modules_vu_prepare_p1 AS
 SELECT 1
 GO
 
+CREATE PROCEDURE sys_all_sql_modules_more_than_4k
```

Comment:
Is there any existing dependency test cases for these views? sql_modules, system_sql_modules, all_sql_modules and all_sql_modules_internal?

Reply:
Yes, we have that in place.

Reply:
sys-sql_modules-dep-*, sys-all_sql_modules-dep-* etc.

Reply:
None of these files depends on "definition" column, one being updated by us. We should have dependency object such that it uses "definition" column

Reply:
Added such tests which uses definition column and uses more than 4k characters